stl/src/xdnorm.cpp

```diff
@@ -8,10 +8,9 @@
 _EXTERN_C_UNLESS_PURE
 
 short _Dnorm(_Dval* ps) { // normalize double fraction
-    short xchar;
+    short xchar         = 1;

```

Comment:
The scope is small enough here as to make no difference, but in the future I suggest moving the declaration to the initialization in preference to moving the initialization to the declaration. We generally try to limit the lifetime of objects to the portion of the code that actually uses those objects.